test/dlc_tests/conftest.py

```diff
@@ -335,14 +332,18 @@ def generate_unique_values_for_fixtures(metafunc_obj, images_to_parametrize, val
 
                     image_tag = image.split(":")[-1].replace(".", "-")
 
-                    framework = image.split(":")[0].split("/")[1].split("-")[0]
-                    job_type = image.split(":")[0].split("/")[1].split("-")[1]
+                    framework_tag = (
+                        "tf" if "tensorflow" in image else "pt" if "pytorch" in image else "mx"
+                        if "mxnet" in image else ""
+                    )
```

Comment:
Please use the `get_framework_from_image_uri()` function to get the image framework.